### PR TITLE
fix: remove appium_apk_tools.jar for macOS notarization

### PR DESCRIPTION
**Pull request by KazuCocoa**
This jar has been removed in https://github.com/appium/appium-adb/pull/471
`appium_apk_tools.jar/prebuilt/aapt/macosx/aapt` in the jar also got `The binary uses an SDK older than the 10.9 SDK` error.

### CODE REVIEW

File: lib/adb.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ class ADB {
 
   initJars () {
     const tempJars = [
-      'move_manifest.jar', 'sign.jar', 'appium_apk_tools.jar',
+      'move_manifest.jar', 'sign.jar',
```
Comment (mykola-mokhnach) [inline lib/adb.js]: I think move_manifest.jar could also be safely excluded

Comment (mykola-mokhnach) [reply lib/adb.js]: it also makes sense to rebuild other jars with newer java version to make the notarization process successful

**Comment by KazuCocoa** [comment]
```
$ git grep "move_manifest.jar"
lib/adb.js:      'move_manifest.jar', 'sign.jar',
$ git grep "move_manifest"
lib/adb.js:      'move_manifest.jar', 'sign.jar',
```

File: lib/adb.js (hunk 1 above)
Comment (KazuCocoa) [reply lib/adb.js]: Yes, but other jars are no issue for the notarization according to my friend.
So, let me do it as another PR

**Pull request merged**